docs/sources/explore/logs-integration.md

```diff
@@ -85,13 +85,13 @@ Set this to True if you want the display to use line wrapping. If set to False,
 
 Set this to `true` to pretty print all JSON logs. This setting does not affect logs in any format other than JSON.
 
-#### Deduping
+#### Deduplicating
```

Comment:
```suggestion
#### Deduplicating
```

As all other headings have the same name as the setting.

Reply:
Shouldn't it be `Deduplication` here as well https://github.com/grafana/grafana/pull/62944/files#diff-6d428f1742c9ebba65767b8d6ea3e6fc55c24978f60dcc96f1dbb782ccd64c2dR441? Would also align with the settings in the logs panel https://github.com/grafana/grafana/blob/main/public/app/plugins/panel/logs/module.tsx#L48